tests/parsers/test.manifestjson.js

```diff
@@ -100,6 +100,29 @@ describe('ManifestJSONParser manifestVersion', function() {
 
 });
 
+describe('ManifestJSONParser bad permissions', function() {
+
+  it('should error if permission is not a string', () => {
```

Comment:
A success case is presumably tested implicitly it might be nice to add a specific test in any case?